### DISCUSSION TITLE
SSH sync

### DISCUSSION THREAD

**Discussion by rswitala** [markshust/docker-magento#555] [Q&A]
Hi,
@markshust could you please write more about this new sync?
I would like to test it, but after starting the new environment with the bin/start command, I only have the nginx.conf file on the SSH container. 
After creating any files in the app container, nothing syncs there, and there is still only that one nginx.conf file. Also I can not upload anything from the host to SSH, because the owner of the html directory is root and not the "app" user.
Maybe I didn't understand something from the description?

Thanks for m1 support! :)

**Comment by markshust** [comment]
Hmmm I think something wonky is going on with your machine. Some things to clarify up front though:
- The SSH image does not do any syncing at all.
- You must use something like PhpStorm's SFTP functionality to set up the sync from the host to the container. Files do not sync back from the container to the host (you must use something like the `bin/copyfromcontainer` command).
- To get it to work, you must copy the `docker-compose.dev-ssh.yml` file to `docker-compose.dev.yml`. My guess is that you copied it over to `docker-compose.yml` instead, which sounds like your results above. The `bin/start` helper script starts up the Docker containers by basically combining those two files together.

**Reply by rndgf** [reply]
Hi @markshust i'm "@doxeone" on twitter, i just ask you a question about this exact same issue.
I do the exact step you describe just above, on a fresh recreated instance : fresh, volumes, containers etc.
(maybe here it's a best place to discuss than on twitter)
